monitoring/oplog-populator/dashboard.py

```diff
@@ -0,0 +1,335 @@
+            'job="${oplog_populator_job}"',
+            'connector=~"${connector}"',
+            'namespace="${namespace}"',
+        ) for name, extraLabels in {
+            'oplog_populator_connector_configuration': ['connector', 'opType'],
+            'oplog_populator_connector_buckets': ['connector'],
+            'oplog_populator_connector_request_bytes': ['connector'],
+            'oplog_populator_connector_pipeline_bytes': ['connector'],
+        }.items()
+    ]
+
+    CONNECTORS = metrics.CounterMetric(
+        'oplog_populator_connectors', 'isOld',
+        'job', 'namespace'
+    ).with_defaults(
+        'job="${oplog_populator_job}"',
+        'namespace="${namespace}"',
+    )
+
+oplog_populator_up_current = Stat(
+    title='Oplog Populator Instances',
+    description='Number of active oplog populator instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='\n'.join([
+            'sum(up{',
+            '    namespace="${namespace}",',
+            '    job="${oplog_populator_job}"',
+            '})',
+        ]),
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+kafka_connect_up_current = Stat(
+    title='Kafka Connect Instances',
+    description='Number of active kafka-connect instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='\n'.join([
+            'sum(up{',
+            '    namespace="${namespace}",',
+            '    job="${kafka_connect_job}"',
+            '})',
+        ]),
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+connector_instances = Stat(
+    title='Active Connectors',
+    description='Number of active kafka-connect connector instances',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='sum(' + Metrics.CONNECTORS.raw() + ')',
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+reused_connector_instances = Stat(
+    title='Reused Connectors',
+    description='Number of active kafka-connect connectors not created by the current oplog populator instance',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
+    reduceCalc='last',
+    graphMode='none',
+    targets=[Target(
+        expr='sum(' + Metrics.CONNECTORS.raw(isOld='true') + ')',
+    )],
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+total_number_buckets = Stat(
+    title='Total Buckets',
+    description='Number of buckets configured on the connectors',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='value',
+    noValue='0',
```

Comment:
no 'noValue' here : if we do not have any metric, best display the default ("-") instead of 0, since we don't know the acutal number...